.github/workflows/build-hw-reusable.yml

```diff
@@ -82,8 +95,9 @@ jobs:
         -DUR_ENABLE_TRACING=ON
         -DUR_DEVELOPER_MODE=ON
         -DUR_BUILD_TESTS=ON
-        -DUR_CONFORMANCE_TEST_LOADER=OFF
         -DUR_BUILD_ADAPTER_${{matrix.adapter.name}}=ON
+        -DUR_CONFORMANCE_TEST_LOADER=${{ matrix.adapter.other_name != '' && 'ON' || 'OFF' }}
```

Comment:
I might be missing something but I don't get why we need to enable another adapter to test the loader? couldn't we just enable it for testing in a job if we need to test it?

Reply:
UR will not use the loader if only one adapter is built. Multiple adapters in the build are required to make it use the loader.

Reply:
Ah, got it. Thanks!